utilities/conv_gefs2datm/conv_gefs2datm.f

```diff
@@ -0,0 +1,888 @@
+C----  convert surface fluxes data from GEFS to DATM
+C----                         hyun-chul.lee@noaa.gov
+C----   12/06/19 change input from prate_avesfc to prateb_avesfc : H-C
+C----   ::: prate_avesfc is daily accumulated precipitation in [mm]
+C----   ::: prateb_avesfc is precipitation rate in [Kg/m^3/s]
+      program conv_gefs2datm
+      implicit none
+      include 'netcdf.inc'
+
+C C This is the name of the data file we will read. 
+      character*(*) FILE_NAME, FOUT_NAME
+      character STR*200,SATT*200
+      parameter (FILE_NAME = "./gefs_input.nc")
+      parameter (FOUT_NAME = "./gefs_output.nc")
+
+C C We are reading 
+      integer NX,NY,NT,nerr,NDIM,i,j,k,NSTR
+      parameter (NX = 1536, NY = 768, NT = 1, NDIM = 3)
+      real*4 :: latitude(NY), longitude(NX)
+      integer time(NT),NF_64BIT_OFFSET,vi_indx
```

Comment:
The NF_64BIT_OFFSET is defined at the netcdf library, it has to be removed from here.